dashboard/db/migrate/20210816152748_update_existing_pd_international_opt_in_countries.rb

```diff
@@ -0,0 +1,39 @@
+# Update existing country values to be english and/or capitalized in
+# order to be consistent with country values in our other tables
+
+class UpdateExistingPdInternationalOptInCountries < ActiveRecord::Migration[5.2]
+  def change
+    english_countries = Set.new(%w(canada chile colombia israel malaysia mexico thailand uzbekistan))
+
+    reversible do |dir|
+      dir.up do
+        ActiveRecord::Base.transaction do
+          # Do I need to batch?
+          Pd::InternationalOptIn.all.each do |opt_in|
```

Comment:
There are about 3500 rows in this table. Let me know if we need to batch to avoid reaching any resource thresholds.